regnety/utils/image_utils.py

```diff
@@ -13,7 +13,8 @@ def is_png(filename):
 
 
 def is_cmyk(filename):
-    blacklist = tf.constant([
+    blacklist = tf.constant(
```

Comment:
Consider a more specific term than `blacklist`, like: `cmyk_filenames`. It's more clear, and avoids potential offense.

As a general note, this kind of change is happening across the industry and is good to become aware of. Some examples:

 * [A recent Rails issue](https://github.com/rails/rails/issues/33677)
 * [Some Chromium changes](https://www.theregister.com/2019/09/03/chromium_microsoft_offensive/)

Reply:
Sorry about that. I'll change it right away.